### ISSUE TITLE
Price Plan Models

### ISSUE THREAD

**Issue by chargio** [project-koku/kokudaka#3]
Define models for Price Plan models for storing Price Plans. Currently won't be backed up by a database.

Please update with implementation details

## Implementation Details

**_Price Plan_**
| Key  | Type|
 | --- | --- |
 | **uuid** | UUIDField; Primary key; read-only |
 | **price_plan_name** | TextField; required |

**_Rate**

| Key  | Type|
 | --- | --- |
 | **uuid** | UUIDField; Primary key; read-only |
 | **price_plan_name** | TextField; required |
 | **price_plan_id** | Integer; required |

**_Rate_**

| Key  | Type|
 | --- | --- |
 | **uuid** | UUIDField; Primary key; read-only |
 | **bucket** | CharField; max_length=63; required  |

**_Tier_**

| Key  | Type|
 | --- | --- |
 | **uuid** | UUIDField; Primary key; read-only |
 | **bucket** | CharField; max_length=63; required  |

**Comment by chambridge** [comment]
Models don't seem to be complete, but copy and pastes from other issues.

**Comment by chargio** [comment]
Yes. It is not complete, need to be updated and copied to the wiki as documentation
